src/app/public/modules/repeater/repeater-item-context-menu.component.ts

```diff
@@ -2,6 +2,9 @@ import {
   Component
 } from '@angular/core';
 
+/**
+ * Wraps and styles a [skyux-dropdown](https://developer.blackbaud.com/skyux-popovers/docs/dropdown) component.
```

Comment:
I'd either format `skyux-dropdown` as a code snippet or change it to just `dropdown` to refer to the component generically rather than by its selector. And either way, I'd pull `component` into the link text.

Reply:
Done.